src/database.rs

```diff
@@ -245,6 +284,12 @@ impl Database {
         Ok(Self { tx })
     }
 
+    pub async fn initialize_server_info(&self) -> Result<String, DbError> {
```

Comment:
why not just do it at db init? We do it every time, there is not much sense to do this in parallel, we do it only once, if this fails we should not be running this system anyway.